### DISCUSSION TITLE
Welcome to the-power-of-prolog Discussions!

### DISCUSSION THREAD

**Discussion by triska** [triska/the-power-of-prolog#34] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JeanChristopheRohner** [comment]
Hi!

I'm Jean-Christophe. I use Prolog for fun and for work, exploring how to induce, evaluate and use scientific theories.

/JCR

**Comment by JeanChristopheRohner** [comment]
@triska I am curious about your thoughts regarding combining inductive logic programming and machine learning :-) In my field (psychology) mainstream data analysis approaches involve traditional machine learning (e.g. multiple regression, support vector machines, neural networks). The advantage with traditional machine learning is that you can induce relations that involve quantitative variables (e.g. force from acceleration and mass). The advantage with ILP is that you can induce _programs_  that have the same expressive power as FOL. Do you have any thoughts about how these approaches can be integrated? I am familiar with probabilistic inductive logic programming and deep learning + logic(e.g. CPLINT, ProbLog, DeepProbLog) but would appreciate your input :-)

**Comment by josd** [comment]
Thanks @triska and Keep Up The Good Work!

I love Prolog since a long time and now within the [KNoWS team](https://knows.idlab.ugent.be/team/) I am trying to use it for [Solid](https://solidproject.org/about) reasoning.
So we currently work on [heisener](https://github.com/josd/heisener) on top of [Scryer Prolog](https://github.com/mthom/scryer-prolog) and this is just a beginning ;-)
